src/master-playlist-controller.js

```diff
@@ -695,11 +706,7 @@ export default class MasterPlaylistController extends videojs.EventTarget {
 
   dispose() {
     this.masterPlaylistLoader_.dispose();
-    for (let loader in this.audioPlaylistLoaders_) {
-      if (this.audioPlaylistLoaders_.hasOwnProperty(loader)) {
-        this.audioPlaylistLoaders_[loader].dispose();
-      }
-    }
```

Comment:
Don't we need to still dispose of audioPlaylistLoaders? I think the active playlistLoader could have a timer (to refresh the playlist on a live hls stream) or an outstanding XHR request that we aren't cleaning up/aborting.